app/scripts/modules/core/src/managed/utils/sortEnvironments.spec.ts

```diff
@@ -0,0 +1,124 @@
+import { IManagedApplicationEnvironmentSummary, IManagedArtifactVersionEnvironment } from 'core/domain';
+import { sortEnvironments } from './sortEnvironments';
+
+type Env = 'test' | 'staging' | 'main';
+
+const getBaseResponse = () => {
+  const versionEnviroments: { [key in Env]: IManagedArtifactVersionEnvironment } = {
+    test: {
+      name: 'test',
+      state: 'current',
+      deployedAt: '2021-04-02T18:13:38.202Z',
+      compareLink:
+        'https://stash.corp.netflix.com/projects/spkr/repos/keel-nflx/compare/commits?targetBranch=2a6f62b40aa24a26d9738d4f6109726cbf284982&sourceBranch=7834ab670dabbd8a0d891ef6b761b7002181164b',
+    },
+    main: {
+      name: 'main',
+      state: 'current',
+      deployedAt: '2021-04-02T18:59:53.108Z',
+      constraints: [
+        {
+          type: 'manual-judgement',
+          status: 'OVERRIDE_PASS',
+        },
+        {
+          type: 'depends-on',
+          status: 'PASS',
+          attributes: { dependsOnEnvironment: 'staging' },
+        },
+      ],
+      compareLink:
+        'https://stash.corp.netflix.com/projects/spkr/repos/keel-nflx/compare/commits?targetBranch=998424c738bf8f2c00586ac5c2d3a4205e1cf80f&sourceBranch=7834ab670dabbd8a0d891ef6b761b7002181164b',
+    },
+    staging: {
+      name: 'staging',
+      state: 'current',
+      deployedAt: '2021-04-02T18:32:03.438Z',
+      constraints: [
+        {
+          type: 'depends-on',
+          status: 'PASS',
+          attributes: { dependsOnEnvironment: 'test' },
+        },
+      ],
+      compareLink:
+        'https://stash.corp.netflix.com/projects/spkr/repos/keel-nflx/compare/commits?targetBranch=2a6f62b40aa24a26d9738d4f6109726cbf284982&sourceBranch=7834ab670dabbd8a0d891ef6b761b7002181164b',
+    },
+  };
+
+  const response: IManagedApplicationEnvironmentSummary = {
+    applicationPaused: false,
+    hasManagedResources: true,
+    resources: [],
+    environments: Object.values(versionEnviroments).map((env) => ({ name: env.name, artifacts: [], resources: [] })),
+    artifacts: [
+      {
+        name: 'keel',
+        type: 'deb',
+        reference: 'keel',
+        versions: [
+          {
+            version: 'v2',
+            displayName: '0.1006.0',
+            environments: Object.values(versionEnviroments),
+            createdAt: '2021-04-02T17:57:58.601Z',
+            lifecycleSteps: [],
+          },
+          {
+            version: 'v1',
+            displayName: '0.1006.0',
+            environments: [versionEnviroments.test, versionEnviroments.staging, versionEnviroments.main],
+            createdAt: '2021-04-02T17:57:58.601Z',
+            lifecycleSteps: [],
+          },
+        ],
+      },
+    ],
+  };
+  return { response, versionEnviroments };
+};
+
+fdescribe('sort environments', () => {
```

Comment:
```suggestion
describe('sort environments', () => {
```

Reply:
I wish there were a way to run a single test via the command line :(